### PR TITLE
Documentation Redesign

### PR DESCRIPTION
**Pull request by saadrahim**

### CODE REVIEW

File: docs/sphinx/how_to/magma_install/magma_install.md [hunk 1]
```diff
@@ -9,4 +9,17 @@ each framework. Refer to the ROCm Compatible Frameworks Release Notes for each
 framework's most current release notes at
 /bundle/ROCm-Compatible-Frameworks-Release-Notes/page/Framework_Release_Notes.html.
 ## PyTorch
-PyTorch is an open source Machine Learning Python library, primarily differentiated by Tensor computing with GPU acceleration and a type-based automatic differentiation. Other advanced features include:
\ No newline at end of file
+PyTorch is an open source Machine Learning Python library, primarily differentiated by Tensor computing with GPU acceleration and a type-based automatic differentiation. Other advanced features include:
+• Support for distributed training
+
+•  Native ONNX support
```
Comment (saadrahim) [inline docs/sphinx/how_to/magma_install/magma_install.md:15]: @Ehud-Sharlin you have some unicode bullets. You should replace with -

Comment (Ehud-Sharlin) [reply docs/sphinx/how_to/magma_install/magma_install.md:15]: Thanks Saad, I know... Just trying out things...
I am experincing long (5 minish) delays between my commits and the time I can see them rendered on https://advanced-micro-devices-rocm--1883.com.readthedocs.build/en/1883/how_to/magma_install/magma_install.html , not sure why...
Anyways, I'll keep working on this! Have a good night

Comment (saadrahim) [reply docs/sphinx/how_to/magma_install/magma_install.md:15]: Build delays of 5 minutes are expected as the ReadTheDocs pipeline is fairly slow. Locally building docs is another option we should explore.

File: docs/sphinx/how_to/magma_install/magma_install.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+The following sections cover the different framework installations for ROCm and
+Deep Learning applications. Figure 5 provides the sequential flow for the use of
+each framework. Refer to the ROCm Compatible Frameworks Release Notes for each
+framework's most current release notes at
+/bundle/ROCm-Compatible-Frameworks-Release-Notes/page/Framework_Release_Notes.html.
+## PyTorch
+PyTorch is an open source Machine Learning Python library, primarily differentiated by Tensor computing with GPU acceleration and a type-based automatic differentiation. Other advanced features include:
+- Support for distributed training
+
```
Comment (saadrahim) [inline docs/sphinx/how_to/magma_install/magma_install.md:14]: The line spacing the original source had is not needed in markdown. We prefer to set the format via central style sheets.

Comment (saadrahim) [reply docs/sphinx/how_to/magma_install/magma_install.md]: Also, forgot to mention these details are not that important. There are automated tools to fix up these files to conform to markdown standards.

File: docs/sphinx/how_to/magma_install/magma_install.md [hunk 3]
```diff
@@ -8,7 +8,8 @@ Deep Learning applications. Figure 5 provides the sequential flow for the use of
 each framework. Refer to the ROCm Compatible Frameworks Release Notes for each
 framework's most current release notes at
 /bundle/ROCm-Compatible-Frameworks-Release-Notes/page/Framework_Release_Notes.html.
-![Figure 5][/figures/image.005.png]
+
+![Figure 5](/figures/image.005.png)
```
Comment (saadrahim) [inline docs/sphinx/how_to/magma_install/magma_install.md:12]: @Ehud-Sharlin Your directory is spelled with capital Figure, please change to lowercase. It is case sensitive.
Also, link should be (figures/image.005.png)

Comment (Ehud-Sharlin) [reply docs/sphinx/how_to/magma_install/magma_install.md]: The vscode markdown preview shows Figure 5, while the readthedocs rendered version doesn't... I'll fix this tomorrow!

**Comment by Ehud-Sharlin** [comment]
Good morning Saad, I am sorry for the bother: I am trying to repeat the cloning/readthedocs process you guided me through, but this time on my laptop, and I think I am missing a step... Can you please send me the steps you followed? (or, only if you have 15 min later today, maybe you can guide me through this again but this time I will document the steps to make sure I include them in the how-to-guide we will provide future team members). Thanks very much!

File: docs/sphinx/how_to/magma_install/magma_install.md (hunk 3 above)
Comment (saadrahim) [reply docs/sphinx/how_to/magma_install/magma_install.md]: Fixed

Comment (Ehud-Sharlin) [reply docs/sphinx/how_to/magma_install/magma_install.md]: Thanks Saad!
A quick question: how should I handle in Markdown the code snippets? (e.g. "docker pull rocm/pytorch:latest"). 
Thanks
(p.s. I will to go offline for a few hours but will get back to this later today)

Comment (saadrahim) [reply docs/sphinx/how_to/magma_install/magma_install.md]: ``` test ```

**Pull request merged**